scripts/baseline_view.py

```diff
@@ -152,11 +153,11 @@ def update_table(self):
     self._table_list = self.table.items()
 
   def gps_time_callback(self, data):
-    self.week = sbp_messages.GPSTime(data).wn
-    self.nsec = sbp_messages.GPSTime(data).ns
+    self.week = MsgGPSTime(data).wn
```

Comment:
I don't have a great sense of the typing here: is `data` the payload of an SBP message or the entire SBP object? If it's only the payload, this won't work, because the constructor takes an entire SBP object.